src/Symfony/Component/Console/Command/Command.php

```diff
@@ -134,6 +134,10 @@ public function __construct(?string $name = null)
             $this->setHelp($attribute?->help ?? '');
         }
 
+        foreach ($attribute?->usages ?? [] as $usage) {
```

Comment:
I don't think it's nullable. Same as https://github.com/symfony/symfony/pull/60767#discussion_r2142558182

```suggestion
        foreach ($attribute->usages as $usage) {
```

Reply:
Oh, `$attribute` can be `null`.

Reply:
`$attribute` itself might be `null` in that class. So the current code is correct.